include/freerdp/server/shadow.h

```diff
@@ -144,6 +144,7 @@ struct rdp_shadow_server
 	UINT32 h264QP;
 
 	char* ipcSocket;
+	char* bindAddress;
```

Comment:
API breakage, not eligible for 2.0. Easy fix could be to reuse `ipcSocket` and set `port` to something `> 65525`